src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -1244,7 +1277,13 @@ private void DisposeTcpClient()
 
         private void OnSuccessfulTwoWaysCommunication() => SuccessfulTwoWaysCommunication?.Invoke(this);
 
-        private void OnSuccessfulReplication() => SuccessfulReplication?.Invoke(this);
+        private void OnSuccessfulReplication()
+        {
+            SuccessfulReplication?.Invoke(this);
+            _parent._server.NotificationCenter.Dismiss(_missingAttachmentsAlert?.Id);
```

Comment:
what if we pass null there? I do not see a code inside Dismiss that checks for that?

Reply:
Done